src/contracts/event_factory.js

```diff
@@ -25,9 +32,12 @@ export async function createTopic(args) {
     return;
   }
 
+  // Break eventName into 10 strings of 64 chars
+  let eventNameStringArray = utils.chunkString(eventName, EVENTNAME_STR_LENGTH).slice(0, EVENTNAME_ARRAY_CAPACITY);
```

Comment:
does the slice need to be `EVENTNAME_ARRAY_CAPACITY - 1`?

Reply:
Did a test with chunkString in jsFiddle.

var str = "12345678901234567890123456789012345678901234567890123456789012345678901234567890";

console.log(chunkString(str, 32));
console.log(chunkString(str, 32)[0].length);

Result:
(3) ["12345678901234567890123456789012", "34567890123456789012345678901234", "5678901234567890"]
32

Reply:
so dont need EVENTNAME_ARRAY_CAPACITY - 1